### ISSUE TITLE
Question on performance gap

### ISSUE THREAD

**Issue by stickyfiner** [qjadud1994/DRS#5]
Hi, Thank you for your code and paper. I've already tried your code, but I cannot achieve the same performance as the paper. Would you please help me figure out where the problem is?
I retrained DeepLabV2 with your shared refined pseudo segmentation labels. But the performance on val only achieves 64.8 and 65.7(after crf). All hyper-parameters follow your provided code. Could you please point out the possible mistake? 
Thank you!

**Comment by qjadud1994** [comment]
Before we uploaded the code, we already reproduced the performance.

We recommend checking the DeepLab-V2 init weight [check it here](https://github.com/qjadud1994/DRS/issues/2)

If you have the same issue after try that, we will reproduce our code again.

Also, for the training recipe of DeepLabV2, we strictly followed the [DeepLab-V2-pytorch](https://github.com/kazuto1011/deeplab-pytorch) code. 
Please check the environmental settings in README.

**Comment by stickyfiner** [comment]
@qjadud1994 Thanks for your reply! I have trained the DeepLabV2 with the provided pseudo labels again following the provided script. The performance on val set is 66.5(67.6 after crf) and 67.0(68.2 after crf) with 2 gpu setting and 4 gpu setting, respectively.  The script is as follows:

DATASET:
    NAME: vocaug
    ROOT: ../PascalVOC2012/VOCdevkit
    LABELS: ./data/datasets/voc12/labels.txt
    N_CLASSES: 21
    IGNORE_LABEL: 255
    SCALES: [0.5, 0.75, 1.0, 1.25, 1.5]
    SPLIT:
        TRAIN: train_cls
        VAL: val_cls
        TEST: test

IMAGE:
    MEAN:
        R: 122.675
        G: 116.669
        B: 104.008
    SIZE:
        BASE: # None
        TRAIN: 321
        TEST: 513

SOLVER:
    BATCH_SIZE:
        TRAIN: 10(20 for 4 gpu setting)
        TEST: 1
    ITER_MAX: 10000
    ITER_SIZE: 2
    ITER_SAVE: 2000
    ITER_TB: 50
    LR_DECAY: 10
    LR: 2.5e-4
    MOMENTUM: 0.9
    OPTIMIZER: sgd
    POLY_POWER: 0.9
    WEIGHT_DECAY: 5.0e-4
    AVERAGE_LOSS: 20
CRF:
    ITER_MAX: 10
    POS_W: 3
    POS_XY_STD: 1
    BI_W: 4
    BI_XY_STD: 67
    BI_RGB_STD: 3

could you please point out the mistake of the script? Thanks a lot!

**Comment by qjadud1994** [comment]
We guess the initial (COCO pretrained) weight is the matter.
When we reproduce our implementation, we got 70.1% mIoU with CRF.

However, DeepLab-V2 without COCO-pretrained weight cannot reproduce their official performance even in the fully supervised setting because of the small batch size issue.

Currently, we newly upload the DeepLab-V3 network which does not require COCO-pretrained weight. please check the details in [here](https://github.com/qjadud1994/DRS#training-the-deeplab-v3-using-pseudo-labels)
